login/login.css

```diff
@@ -0,0 +1,235 @@
+body {
+  margin: 0;
+  overflow: hidden;
+}
+
+.main {
+  min-height: 100%;
+  margin: 0 auto;
+  width: 100%;
+  display: flex;
+  align-content: center;
+  align-items: center;
+  justify-content: center;
+  
+}
+
+.bg {
+  background-image: url(images_login/bg.png);
+  -webkit-background-size: contain;
+  background-size: cover;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  width: 100%;
+  
+ 
+}
+
+.bg.no-image {
+  background-image: none;
+}
+
+.bg:after {
+  content: ' ';
+  height: 100%;
+  position: absolute;
+  width: 100%;
+  opacity: .5;
+  background: linear-gradient(to bottom right, #FF6A3A 0%, #000000 100%);

```

Comment:
the gradient is from bottom to top, don't need for 0% and 100% , those are the predefined ones , also i think that it should be .7 opacity to look more like in the design